Games/types/Mafia/roles/cards/PotionCaster.js

```diff
@@ -0,0 +1,152 @@
+const Card = require("../../Card");
+const { PRIORITY_KILL_DEFAULT, PRIORITY_NIGHT_SAVER, PRIORITY_INVESTIGATIVE_DEFAULT } = require("../../const/Priority");
+
+module.exports = class PotionCaster extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
+            "Cast Potion": {
+                states: ["Night"],
+                flags: ["voting"],
+                action: {
+                    priority: PRIORITY_NIGHT_SAVER - 1,
+                    run: function () {
+                        // set target
+                        this.actor.role.data.currentTarget = this.target;
+
+                        // set cooldown
+                        var potion = this.actor.role.data.currentPotion;

```

Comment:
the cooldown should be set individually in each action to prevent commuter blocks using up a potion's cooldown